public/app/plugins/panel/nodeGraph/module.tsx

```diff
@@ -39,4 +46,150 @@ export const plugin = new PanelPlugin<NodeGraphOptions>(NodeGraphPanel).setPanel
       });
     },
   });
+  builder.addNestedOptions({
+    category: ['Node names'],
```

Comment:
Would probably give a different name for the section like "Node field name overrides". It is a bit lengthy but otherwise, there is not much to indicate what this section is about.